conda/meta.yaml

```diff
@@ -53,7 +53,7 @@ requirements:
     - python
     - setuptools
     - setuptools_scm
-    - numpy =1.20
+    - numpy >=1.20
```

Comment:
Consider ~=1.20. I think that means "any version that should be compatible with 1.20"